### ISSUE TITLE
Gnu/Linux install script has mispelled ASCII art

### ISSUE THREAD

**Issue by Erudition** [laurent22/joplin#1212]
in the install script `Joplin_install_and_update.sh`:

```
       _             _         _           _        _ _           
      | |           (_)       (_)         | |      | | |          
      | | ___  _ __  _ _ __    _ _ __  ___| |_ __ _| | | ___ _ __ 
  _   | |/ _ \\| '_ \| | '_ \  | | '_ \\/ __| __/ _` | | |/ _ \ '__|
 | |__| | (_) | |_) | | | | | | | | | \__ \ || (_| | | |  __/ |   
  \____/ \___/| .__/|_|_| |_| |_|_| |_|___/\__\__,_|_|_|\___|_|   
              | |                                                 
              |_|   
```

Hmm, there seems to be a stray slash. Let's remove that:

```
       _             _         _           _        _ _           
      | |           (_)       (_)         | |      | | |          
      | | ___  _ __  _ _ __    _ _ __  ___| |_ __ _| | | ___ _ __ 
  _   | |/ _ \| '_ \| | '_ \  | | '_ \\/ __| __/ _` | | |/ _ \ '__|
 | |__| | (_) | |_) | | | | | | | | | \__ \ || (_| | | |  __/ |   
  \____/ \___/| .__/|_|_| |_| |_|_| |_|___/\__\__,_|_|_|\___|_|   
              | |                                                 
              |_|   
```

Yup, that definitely says "Jopin installer". What's "Jopin"? :P

**Comment by tessus** [comment]
This can be fixed rather easily ;-):

```
       __            ___          _            __        ____         
      / /___  ____  / (_)___     (_)___  _____/ /_____ _/ / /__  _____
 __  / / __ \/ __ \/ / / __ \   / / __ \/ ___/ __/ __ `/ / / _ \/ ___/
/ /_/ / /_/ / /_/ / / / / / /  / / / / (__  ) /_/ /_/ / / /  __/ /    
\____/\____/ .___/_/_/_/ /_/  /_/_/ /_/____/\__/\__,_/_/_/\___/_/     
          /_/                                                         

or:

       _             _ _         _           _        _ _           
      | |           | (_)       (_)         | |      | | |          
      | | ___  _ __ | |_ _ __    _ _ __  ___| |_ __ _| | | ___ _ __ 
  _   | |/ _ \| '_ \| | | '_ \  | | '_ \/ __| __/ _` | | |/ _ \ '__|
 | |__| | (_) | |_) | | | | | | | | | | \__ \ || (_| | | |  __/ |   
  \____/ \___/| .__/|_|_|_| |_| |_|_| |_|___/\__\__,_|_|_|\___|_|   
              | |                                                   
              |_|                                                   

```

Now someone has to create a PR...

**Comment by tessus** [comment]
fixed in 99156311db8bb17c8e31cc40378bf858c594f05c
